packages/vscode-extension/src/debug/launch.ts

```diff
@@ -1,7 +1,14 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import { UserError } from "@microsoft/teamsfx-api";
+import open = require("open");
```

Comment:
Can this package be imported using "import from"?

Reply:
No, it requires `esModuleInterop` to be true in `tsconfig.json`.